### ISSUE TITLE
Piped proxy quick toggle

### ISSUE THREAD

**Issue by codenyte** [libre-tube/LibreTube#6089]
### Describe your suggested feature

Piped instances that don't use IPv6 rotation (https://docs.piped.video/docs/self-hosting/#ipv6-rotator-using-docker) often get blocked by YouTube, as can be seen in #5855. I think there should be a quick toggle in the player to easily enable/disable the Piped proxy, similar to the SponsorBlock toggle.

### Other details

 I think there should be an option in the settings to hide the toggle, to prevent it from being triggered accidentally.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
That's not planned, many people are self-hosting instances for a small amount of users or maintain reliable instances with IPv6 rotator or a VPN and don't need that button.

Making such a button optional is not a good idea since our settings interface already is too complicated, and the aim is rather to remove very trivial preferences instead of creating new ones.
